src/modules/base.ts

```diff
@@ -1,119 +1,137 @@
 /**
  * Base module for all API modules
  *
- * @module modules/base
- *
  * @since 1.0.0
  */
 import commsUtils from '../utils/comms';
-import {name as PROTOCOL_AGENT_NAME, version as PROTOCOL_AGENT_VERSION} from '../../package.json';
 
 const PROTOCOL_NAME = 'purecloud-client-apps';
 
-const ACTION_NAME_KEY = 'action';
-const PROTOCOL_NAME_KEY = 'protocol';
-const PROTOCOL_AGENT_NAME_KEY = 'protocolAgentName';
-const PROTOCOL_AGENT_VERSION_KEY = 'protocolAgentVersion';
+interface ProtocolDetails {
+    name: string;
+    agentName: string;
+    agentVersion: string;
+}
+
+export interface SDKMessagePayload {
```

Comment:
Thoughts on splitting this interface into and inbound and outboud message payload?

Reply:
I guess the inbound also technically has a property, `purecloudEventType`, but, if we're not casting to this type and using it, probably not necessary.